web/src/app/workplace/components/task-information/task-information.component.html

```diff
@@ -0,0 +1,193 @@
+        </mat-form-field>
+        <div *ngIf="inputOverflowMap.get(company.name)" class="error">{{lengthError}}</div>
+        <taskana-shared-field-error-display [displayError]="!isFieldValid('task.primaryObjRef.company')"
+                                            [validationTrigger]="this.toggleValidationMap.get('task.primaryObjRef.company')"
+                                            errorMessage="* Company is required">
+        </taskana-shared-field-error-display>
+
+
+        <!-- TASK SYSTEM INSTANCE -->
+        <mat-form-field appearance="outline">
+          <mat-label>System Instance</mat-label>
+          <label for="task.primaryObjRef.systemInstance"></label>
+          <input matInput type="text" required maxlength="32" #systemInstance="ngModel" id="task.primaryObjRef.systemInstance" placeholder="System Instance description"
+                 [(ngModel)]="task.primaryObjRef.systemInstance" name="task.primaryObjRef.systemInstance" (input)="validateInputOverflow(systemInstance, 32)">
+        </mat-form-field>
+        <div *ngIf="inputOverflowMap.get(systemInstance.name)" class="error">{{lengthError}}</div>
+        <taskana-shared-field-error-display [displayError]="!isFieldValid('task.primaryObjRef.systemInstance')"
+                                            [validationTrigger]="this.toggleValidationMap.get('task.primaryObjRef.systemInstance')"
+                                            errorMessage="* System instance is required">
+        </taskana-shared-field-error-display>
+
+        <!-- REFERENCE VALUE -->
+        <mat-form-field appearance="outline">
+          <mat-label>Reference value</mat-label>
+          <label for="task.primaryObjRef.value"></label>
+          <input matInput type="text" required maxlength="32" #value="ngModel" id="task.primaryObjRef.value" placeholder="Reference value"
+                 [(ngModel)]="task.primaryObjRef.value" name="task.primaryObjRef.value" (input)="validateInputOverflow(value, 32)">
+        </mat-form-field>
+        <div *ngIf="inputOverflowMap.get(value.name)" class="error">{{lengthError}}</div>
+        <taskana-shared-field-error-display [displayError]="!isFieldValid('task.primaryObjRef.value')"
+                                            [validationTrigger]="this.toggleValidationMap.get('task.primaryObjRef.value')"
+                                            errorMessage="* Reference value is required">
+        </taskana-shared-field-error-display>
+
+
+        <div class="task-information__due-date-and-priority">
+
+          <!-- DUE DATE -->
+          <div class="task-information__due-date">
+            <span style="color: gray; font-size: 0.8em;">Due date</span>
+            <div class="task-information__date-picker" style="position: relative; top: 4px;">
+              <taskana-shared-date-picker placeholder="Due date" [value]="task.due" [name]="'task.due'" [id]="'task-due'"
```

Comment:
I would replace the old datepicker with angular material https://material.angular.io/components/datepicker/overview

Reply:
Yes, I'll do this. Should we keep `taskana-shared-date-picker` and replace the code in `taskana-shared-date-picker` or would you delete the component?

Reply:
you can delete the component